judge/models/problem.py

```diff
@@ -271,6 +273,7 @@ def get_editable_problems(cls, user):
             return cls.objects.all()
 
         q = Q(authors=user.profile) | Q(curators=user.profile)
+        q |= Q(is_organization_private=True, organizations__in=user.profile.organizations.filter(admins=user.profile))
```

Comment:
This feels overly convoluted. Are you sure `user.profile.admin_of.all()` doesn't work?

Reply:
Ah I forgot that was a thing :blobderpy: